firefox-ios/Client/Frontend/Browser/BrowserViewController/Extensions/BrowserViewController+TabToolbarDelegate.swift

```diff
@@ -328,8 +328,8 @@ extension BrowserViewController: ToolBarActionMenuDelegate {
                                     theme: themeManager.currentTheme) { [weak self] isButtonTapped in
                 guard let strongSelf = self, let currentTab = strongSelf.tabManager.selectedTab else { return }
```

Comment:
```suggestion
                guard let self, let currentTab = tabManager.selectedTab else { return }
```

If you do this then you can remove `strongSelf` in this closure . I know it's not your change, I know, but figured might as well clean it up :)